scripts/templates/train-template.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
rename to reflect slurm so like `slurm-train-template.sh`

Reply:
done